src/main/java/de/zalando/aruha/nakadi/domain/Cursor.java

```diff
@@ -0,0 +1,57 @@
+package de.zalando.aruha.nakadi.domain;
+
+public class Cursor {
```

Comment:
At first I wanted to do that but I faced deserialization problems (as there were no setters) which I was lazy to fix :)
Now I made it immutable and fixed deserialization.